### PR TITLE
Add ability to set maximum connection time

### PR DESCRIPTION
**Pull request by JakubOnderka**
And also add current user count to metrics.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -209,7 +209,7 @@ impl App {
             Event::Query(event::Query::Metrics) => match self.redis.connect().await {
                 Ok(mut redis) => {
                     if let Err(e) = redis
-                        .set(
+                        .lpush(
```
Comment (icewind1991) [inline src/lib.rs:212]: What is the reasoning for switching to a list here?

Comment (JakubOnderka) [reply src/lib.rs:212]: Because then it is possible to use blocking fetch `[brpop](https://redis.io/commands/brpop/)` instead of waiting specified time.

Comment (icewind1991) [reply src/lib.rs:204]: Ok, one problem is that if the list doesn't get pop'd for any reason you end up growing the list forever.

Adding an `trim` after the push here should resolve that issue

**Comment by JakubOnderka** [comment]
@icewind1991 I drop the commit that change the way how stats are fetched from Redis for now.

So I think now it is mergable.

**Review by icewind1991 — APPROVED** [review]

**Comment by icewind1991** [comment]
Thanks, feel tree to open another PR with the metrics polling changes if you feel like it.

**Pull request merged**